helm/idol-nifi/values.yaml

```
@@ -239,3 +256,55 @@ idol-licenseserver:
 metrics-server:
   args:
     - "--kubelet-insecure-tls"
+
+prometheus:
+  enabled: true
+  server:
+    prefixURL: /prometheus
+    baseURL: /prometheus
+    ingress:
+      enabled: true
+      hosts:
+        - ""
+      pathType: Prefix
+      path: /prometheus
+    global:
+      scrape_interval: 15s
+
+prometheus-adapter:
+  enabled: true
+  metricsRelistInterval: 30s
+  prometheus:
+    url: http://idol-nifi-prometheus-server
```

```suggestion
    url: http://{{ .Release.Name }}-prometheus-server
```
Needs to match how the prometheus service will be named. ['Value is templated'](https://github.com/prometheus-community/helm-charts/blob/c45f8fb79f8708250b8a9d0bb417a7abc97229c0/charts/prometheus-adapter/values.yaml#L35) according to the chart docs.